library/src/include/utility.h

```diff
@@ -103,6 +103,22 @@ __forceinline__ __device__ __host__ T*
     return p[block] + offset;
 }
 
+// For device pointers
```

Comment:
The above functions can be removed right, since these are replacing them? Duplication probably happened during a merge.

Reply:
good catch